src/material/timepicker/timepicker.md

```diff
@@ -1 +1,310 @@
+      <td><a href="https://moment.github.io/luxon/">See project for details</a></td>
+    </tr>
+    <tr>
+      <th align="left" scope="row">Dependencies</th>
+      <td><a href="https://moment.github.io/luxon/">Luxon</a></td>
+    </tr>
+    <tr>
+      <th align="left" scope="row">Import from</th>
+      <td><code>@angular/material-luxon-adapter</code></td>
+    </tr>
+  </tbody>
+</table>
+
+`provideMomentDateAdapter` or `MatMomentDateModule` (installed via `ng add @angular/material-moment-adapter`)
+
+<table>
+  <tbody>
+    <tr>
+      <th align="left" scope="row">Date type</th>
+      <td><code>Moment</code></td>
+    </tr>
+    <tr>
+      <th align="left" scope="row">Supported locales</th>
+      <td><a href="https://github.com/moment/moment/tree/develop/src/locale">See project for details</a></td>
+    </tr>
+    <tr>
+      <th align="left" scope="row">Dependencies</th>
+      <td><a href="https://momentjs.com/">Moment.js</a></td>
+    </tr>
+    <tr>
+      <th align="left" scope="row">Import from</th>
+      <td><code>@angular/material-moment-adapter</code></td>
+    </tr>
+  </tbody>
+</table>
+
+**Note**: `provideNativeDateAdapter` implements time parsing using a regex which means that it
+only supports AM/PM time (e.g. `1:45 PM`) or 24-hour time (e.g. `22:45` or `22.45`). As such
+it won't work on locales with different formatting. We recommend using one of the provided date
+adapters mentioned above or creating your own adapter by extending the `DateAdapter` class from
+`@angular/material/core`. For example, if you want to use the `date-fns` adapter, you can update
+your `bootstrapApplication` format to the following:
+
+```ts
+import {provideDateFnsAdapter} from '@angular/material-date-fns-adapter';
+
+bootstrapApplication(MyApp, {
+  providers: [provideDateFnsAdapter()]
+});
+```
+
+#### Customizing the parse and display formats
+
+The `MAT_DATE_FORMATS` object is a collection of formats that the timepicker uses when parsing
+and displaying date objects. These formats are passed through to the `DateAdapter` so you will want
+to make sure that the format objects you're providing are compatible with the `DateAdapter` used in
+your app.
+
+`MAT_DATE_FORMATS` is the same object used by `mat-datepicker` so it's likely already
+configured if your app is using the datepicker, but for the timepicker you need to ensure that the
+`display.timeInput`, `display.timeOptionLabel` and `parse.timeInput` properties are set as well.
+
+If you want use one of the `DateAdapters` that ships with Angular Material, but use your own
+`MAT_DATE_FORMATS`, you can either pass the formats into the providers function, or provide the
+`MAT_DATE_FORMATS` token yourself. For example:
+
+```ts
+bootstrapApplication(MyApp, {
+  providers: [provideNativeDateAdapter(MY_NATIVE_DATE_FORMATS)],
+});
+```
+
+### Accessibility
+
+The timepicker implements the ARIA combobox interaction pattern. The timepicker input specifies
```

Comment:
Maybe add a link for ARIA combobox, probably https://www.w3.org/WAI/ARIA/apg/patterns/combobox/